src/Microsoft.TestPlatform.Common/Filtering/FilterExpression.cs

```diff
@@ -50,7 +50,7 @@ internal class FilterExpression
 
         #region Constructors
 
-        internal FilterExpression()
+        private FilterExpression()
```

Comment:
I think we can get rid of this.

Reply:
Replaced with `FilterExpression(FilterExpression left, FilterExpression right, bool areJoinedByAnd)`